SU-SOE/stanford_soe_helper
# 7.x 2.x more links

#  READY FOR REVIEW

# Summary
- This adds views displays for displaying a button link at the bottom of a spotlight node.

# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
For spotlight nodes verify for  Faculty, Office of the Dean, Staff, Students, and Alumni  affiliations
- verify the appropriate "see more" block displays.
- verify that the URL is correct for each link within the block

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
- Work: https://stanfordits.atlassian.net/browse/SOE-2172
- PR: https://stanfordits.atlassian.net/browse/SOE-2218 
## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
